thunx-dependencies/build.gradle

```
@@ -9,8 +9,8 @@ javaPlatform {
 dependencies {
     api platform(project(":thunx-bom"))
 
-    api platform("org.springframework.boot:spring-boot-dependencies:2.7.13")
-    api platform("org.springframework.cloud:spring-cloud-dependencies:2021.0.8")
+    api platform("org.springframework.boot:spring-boot-dependencies:3.0.4")
+    api platform("org.springframework.cloud:spring-cloud-dependencies:2022.0.1")
```

Any reason why you did not select latest patch releases of the `3.0.x` branch ?

```suggestion
    api platform("org.springframework.boot:spring-boot-dependencies:3.0.9")
    api platform("org.springframework.cloud:spring-cloud-dependencies:2022.0.4")
```